test/index.html

```diff
@@ -5,11 +5,12 @@
 <link rel="stylesheet" href="../build/test.css">
 <style>
   .fixture-Box {
-    background: rgb(100, 150, 50);
-    background: rgb(100, 150, 50);
-    border: 1px solid black;
+    background: rgba(100, 150, 50, .75);
+    border: 1px solid #888;
+    font-family: sans-serif;
     margin-bottom: 10px;
```

Comment:
So on reflection I don't think the grid should introduce any vertical spacing. It would break sites that used it for a content/sidebar layout if the bottom of their cells suddenly gained 20px. I've managed to drop this updated Grid into my blog and it all renders the same without any changes so i think that's a good goal to keep.